blobfuse/blobfuse.cpp

```diff
@@ -125,6 +133,11 @@ int read_config(std::string configFile)
             std::string accountKeyStr(value);
             str_options.accountKey = accountKeyStr;
         }
+        else if(line.find("sasToken") != std::string::npos)
```

Comment:
can you please replace commented out code before and after this line:
 /*            if(!is_lowercase_string(accountNameStr))
...*/

Reply:
Do you mean delete?

Reply:
Yeah, that code didn't work... I left it in b/c I thought it might be a good idea to keep it, but client-side checks like these can come back to bite us, if something changes on the service.  Let's just remove it.